win32ss/user/winsrv/usersrv/shutdown.c

```diff
@@ -422,34 +422,29 @@ ConioConsoleCtrlEventTimeout(DWORD Event, PCSR_PROCESS ProcessData, DWORD Timeou
 /************************************************/
 
 
-static VOID
+static BOOL
 ThreadShutdownNotify(IN PCSR_THREAD CsrThread,
                      IN ULONG Flags,
                      IN ULONG Flags2,
                      IN PNOTIFY_CONTEXT Context)
 {
     HWND TopWnd = NULL;
+    HWND hWndOwner;
+    HWND tmpWnd;    /*** FOR TESTING PURPOSES ONLY!! ***/
 
     EnumThreadWindows(HandleToUlong(CsrThread->ClientId.UniqueThread),
                       FindTopLevelWnd, (LPARAM)&TopWnd);
-    if (TopWnd)
-    {
-        HWND hWndOwner;
+    if (TopWnd == NULL)
```

Comment:
Can you split your commit in two: first one that does the code formatting (such as fixing whitespace/indentation level as here) and the second for the code changes?

Reply:
sure